btm/src/test/java/bitronix/tm/mock/AbstractMockJdbcTest.java

```diff
@@ -92,12 +95,6 @@ protected void setUp() throws Exception {
         registerPoolEventListener(p2);
 
         TransactionManagerServices.getConfiguration().setGracefulShutdownInterval(2);
-
-        // start TM
-        TransactionManagerServices.getTransactionManager();
-
-        // clear event recorder list
-        EventRecorder.clear();
     }
```

Comment:
Removed the call to getTransactionManager() to make it easy to alter configuration in testcases.  This also meant we needed to control the EventRecorder in each, but that felt more natural to me anyway since those tests expect very fine grain control over the activity in the ER.